invokeai/frontend/web/src/features/nodes/components/Flow.tsx

```diff
@@ -67,6 +69,10 @@ export const Flow = () => {
     dispatch(connectionEnded());
   }, [dispatch]);
 
+  const onInit: OnInit = useCallback((v: ReactFlowInstance) => {
+    if (v) v.fitView();
```

Comment:
Pls use full conditional syntax eg `if (...) {...}` - it's way too easy for bugs to creep in without the curly braces

Reply:
Curly brace lover I see. Fixed.

Reply:
Yes :P I've borked myself too many times with the more compact syntax!